frontend/pages/admin/manage/groups/_id.vue

```diff
@@ -74,6 +74,10 @@ export default defineComponent({
 
       const { response, data } = await adminApi.groups.updateOne(group.value.id, group.value);
       if (response?.status === 200 && data) {
+        if (group.value.slug !== data.slug) {
+          // the slug updated, which invalidates the nav URLs
+          window.location.reload();
+        }
         group.value = data;
       }
```

Comment:
```suggestion
      if (response?.status === 200 && data) {
        if (group.value.slug !== data.slug) {
          // the slug updated, which invalidates the nav URLs
          window.location.reload();
        }
        group.value = data;
      }
      else {
        alert.error(i18n.tc("settings.settings-update-failed"));
      }
```

Maybe just add that, so the user has some sort of feedback